### PR TITLE
Docusaurus version upgrade to 2.4.1

### PR DESCRIPTION
**Pull request by ghost**
I updated to docusaurus v 2.4.1 Please check and comment if there is any fix requires. Thank you

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -48,5 +49,8 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "dependencies": {
+    "docusaurus": "^1.14.7"
```
Comment (phated) [inline package.json]: Is this the old version?

Comment (ghost) [reply package.json]: @phated  Actually it is recent version. https://www.npmjs.com/package/docusaurus
library with @docusaurus has updated to 2.4.1

**Comment by phated** [comment]
@donghwipark have you run the new changes against the website? I'm seeing a lot of differences. For example, the navbar isn't pinned to the top of the window when you scroll any more.

**Comment by ghost** [comment]
@phated I couldn't realize that defect. I will compare with current web and please let me know defects you realised. Will fix and re-commit.

File: package.json (hunk 1 above)
Comment (phated) [reply package.json]: This package is old and should be deprecated. The new package is https://www.npmjs.com/package/@docusaurus/core which you've updated

**Comment by ghost** [comment]
@phated I fixed the CSS and pushed, please have a look and give me any comments if you need. Thank you

**Comment by ghost** [comment]
I found some changes on text size after the upgrade. Seems default css has changed will update and commit

**Comment by ghost** [comment]
@phated fixed some css after comparing between the new version and current version.

**Comment by phated** [comment]
@donghwipark that's awesome! Thanks for inspecting. Will you also please update the other dependencies that have newer versions? I noticed that `docusaurus-plugin-sass` and other packages are very outdated.

Also, please regenerate and commit the package-lock.json using `npm i -S`

**Comment by ghost** [comment]
@phated Sure Thank you for the guide!

**Comment by phated** [comment]
Replaced by #149

**Pull request closed without merging**